drivers/client_access_control/extensions/grda_warehouse/hud/client_extension.rb

```diff
@@ -134,7 +134,7 @@ def enrollments_for_rollup(user:, en_scope: scope, include_confidential_names: f
             else
               cocs = ''
               if ::GrdaWarehouse::Config.get(:expose_coc_code)
-                cocs = entry.enrollment&.enrollment_cocs&.map(&:CoCCode)&.uniq&.join(', ')
+                cocs = project.project_cocs&.pluck(GrdaWarehouse::Hud::ProjectCoc.coc_code_coalesce)&.reject(&:blank?)&.uniq&.join(', ')
```

Comment:
Hmm, so this is interesting because this is a fundamental change.  I _think_ we want to do it, but I'll need to think it through.